README.md

```diff
@@ -9,3 +9,27 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+If we have two Completely connected graphs with the same number of verticies then it doesn't matter how we map the graphs together we will always be able to map every vertice to every other vertice.
```

Comment:
...but I can map all nodes in the first graph to a single node in the second graph.